packages/sync-actions/test/product-sync-prices.spec.js

```diff
@@ -640,7 +803,7 @@ describe('Actions', () => {
               centAmount: 6495,
               currencyCode: 'EUR',
             },
-            custom: {
+            customField: {
```

Comment:
This looks wrong, please revert this change.

Reply:
done